### PR TITLE
Update ar.toml -Enhance Arabic User Experience with Feedback Labels and Correct Diacritics

### PR DESCRIPTION
**Pull request by jadmadi**
This pull request introduces Arabic feedback labels with accurate diacritics (Tashkeel) to improve the overall user experience for Arabic speakers.

**Key changes:**

- Added Arabic strings for the "feedback" label.
- Corrected and applied mandatory diacritics to Arabic characters.
- Implemented the approved Arabic translation of "feedback" as defined by The Digital Arabic Language Hub (https://www.m-a-arabia.com/vb/showthread.php?t=30619).

This update ensures consistency with Arabic language standards and enhances the clarity and readability of the user interface for Arabic-speaking users.

### CODE REVIEW

**Review by deining — APPROVED** [review]
LGTM

File: i18n/ar.toml [hunk 1]
```diff
@@ -1,80 +1,91 @@
 # UI strings. Buttons and similar.
 
 [ui_pager_prev]
-other = "السابق"
+other = "السَّابق"
```
Comment (chalin) [inline i18n/ar.toml:4]: Removing trailing blank space in **this and all other lines**, for example:

```suggestion
other = "السَّابق"
```

File: i18n/ar.toml [hunk 2]
```diff
@@ -1,80 +1,91 @@
...
+other = "اَلْعَودة لِلْعرْض العاديِّ" 
 [print_entire_section]
-other = "اطبع القسم كاملا"
+other = "اِطْبع القسْم كاملا" 
 
-# Community
+# Community 
 [community_join]
-other = "انضم الي مجتمع {{ .Site.Title }}"
+other = "اِنْضمَّ إِلى مُجتَمَع { { . Site . Title } }"
```
Comment (chalin) [inline i18n/ar.toml:66]: Why all the extra spaces here? Remove the extra spaces added around the periods and braces, here and in other lines:

```suggestion
other = "اِنْضمَّ إِلى مُجتَمَع {{.Site.Title }}"
```

@deining ☝️

File: i18n/ar.toml [hunk 3]
```diff
@@ -1,80 +1,91 @@
...
-other = "إذا كنت ترغب في المشاركة بشكل أكبر من خلال المساهمة في {{ .Site.Title }}, انضم إلينا هنا:"
+other = "إِذَا كُنْت تَرغَب فِي المشاركة بِشَكل أَكبَر مِن خِلَال المساهمة فِي { { . Site . Title } } , اِنْضمَّ إِليْنَا هُنَا :" 
 [community_how_to]
-other = "يمكنك معرفة كيفية المساهمة في {{ .Site.Title }} من خلال"
+other = "يُمْكِنك مَعرِفة كَيفِية المساهمة فِي { { . Site . Title } } مِن خِلَال" 
 [community_guideline]
-other = "إرشادات المساهمة"
+other = "إِرْشادات المساهمة"
+
+
```
Comment (chalin) [inline i18n/ar.toml:82]:
```suggestion

```

File: i18n/ar.toml [hunk 4]
```diff
@@ -1,80 +1,91 @@
...
-other = "كافة الحقوق محفوظة"
+other = "كَافَّة اَلحُقوق مَحفُوظة" 
 
 [footer_privacy_policy]
-other = "سياسة الخصوصية"
-
+other = "سِياسة الخصوصيَّة" 
 
-# Post (blog, articles etc.)
+# Post ( blog , articles etc . )
```
Comment (chalin) [inline i18n/ar.toml:30]:
```suggestion
# Post (blog, articles etc.)
```

**Review by chalin — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please remove the extra spaces added throughout the file, since in the case of `{{...}}` directives, it will break the rendering.

File: i18n/ar.toml (hunk 2 above)
Comment (jadmadi) [reply i18n/ar.toml:66]: I think I'll have to review my editor settings. Will resolve it and update the request.

**Review by chalin — APPROVED** [review]
Thank you!

**Comment by jadmadi** [comment]
@deining Would appreciate reviewing and approving it.

**Review by deining — APPROVED** [review]
LGTM

**Comment by jadmadi** [comment]
This is a stalled pull request. Should I tag someone, or is it just the regular procedure?

**Pull request merged**